bitnami/external-dns/templates/service.yaml

```
@@ -30,6 +30,9 @@ spec:
   {{- if .Values.service.externalIPs }}
   externalIPs: {{ toYaml .Values.service.externalIPs | nindent 4 }}
   {{- end }}
+  {{- if .Values.service.externalName }}
```

This new value only makes sense when [`service.type=ExternalName`](https://kubernetes.io/docs/concepts/services-networking/service/#externalname), I think we should add a condition like this:
```suggestion
  {{- if and (eq .Values.service.type "ExternalName") .Values.service.externalName }}
```

---

Thank you for your review! You are right, I have applied your suggestion. Thanks.